Good Coding Practices/src/css/functionalities.css

```diff
@@ -0,0 +1,31 @@
+#functionalities-wrapper {
+    position: absolute;
+    top: 140px;
+    left: 1%;
+    text-align: center;
+}
+
+#btn-reset {
+    font-size: 30px;
+    border: 5px solid #0F64FF;
+    background: #FFD505;
+    border-radius: 7px;
+    font-family: david;
+}
+#btn-reset:hover {
+    background-color: #F3D7FC;
+    font-size: 31px;
+}
+
+
+#btn-randomize {
+    font-size: 30px;
+    border: 5px solid #FFBC05;
+    background: #05E8C2;
+    border-radius: 5px;
+    font-family: david;
```

Comment:
Better specify `font-family` at a higher level as it applies for all the document